driver/kubernetes/driver.go

```diff
@@ -105,8 +104,8 @@ func (d *Driver) wait(ctx context.Context) error {
 		depl *appsv1.Deployment
 	)
 
-	timeoutChan := time.After(d.timeout)
-	ticker := time.NewTicker(100 * time.Millisecond)
+	timeoutChan := time.After(d.Timeout)
+	ticker := time.NewTicker(d.Timeout / time.Microsecond)
```

Comment:
I'd like to propose quite similar logic here. With the default timeout of 120s we say that every 120ms we poll for replicas. Current behavior is to fail after 120s, with 100ms intervals.

Reply:
What's your opinion? Should the default timeout value be coupled to retry intervals?